packages/oauth/src/index.ts

```diff
@@ -315,12 +315,17 @@ export class InstallProvider {
   ): Promise<void> {
     let parsedUrl;
     let code: string;
+    let flowError: string;
     let state: string;
     let installOptions: InstallURLOptions;
 
     try {
       if (req.url !== undefined) {
         parsedUrl = new URL(req.url);
+        flowError = parsedUrl.searchParams.get('error') as string;
+        if (flowError === 'access_denied') {
+          throw new AuthorizationError('User cancelled the OAuth installation flow!');
```

Comment:
I decided to reuse an existing error, `AuthorizationError`, to describe this situation. Let me know if something else is desired.